src/main/resources/default/assets/wondergem/wondergem.js.pasta

```diff
@@ -21,3 +21,7 @@
 <i:include name="/assets/wondergem/bootstrap-tokenfield/bootstrap-tokenfield.js"/>
 
 <i:include name="/assets/wondergem/autocomplete/multiselect.js"/>
+
+<i:if test="@call.getUserAgent().contains('Trident')">
```

Comment:
nope, this will most definitely be cached by an upstream proxy - move this to the page template and load on demand there

Reply:
moved to w:page